drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -700,6 +701,8 @@ local function do_bridge_network_init(driver, device, bridge_url, api_key)
       log.info_with({ hub_logs = true },
         string.format("Event Source Connection for Hue Bridge \"%s\" established, marking online", device.label))
       device:online()
+      --TODO should we do a refresh of all child devices when we first connect?
+      --handlers.refresh_handlers(driver, device)
```

Comment:
Child devices are "refreshed" on the first join, but not on driver restart. Its not clear to me if we should make the change since that would be a relatively large change to driver restart flow if we are trying to get the fix out in time for 0.49.8 prod release. However, such a change may be required to fix this issue for all cases due to the way the lan child device health feature works with 0.49.8.

Reply:
if we don't do that fix, doesn't this not fix the problem (i.e. child devices remaining offline)?

Reply:
Yup, you're right. Without the change any time a bridge comes online after being offline the children would erroneously remain offline rather than being queried to see if they are online.